internal/core/application/walletunlocker_service.go

```diff
@@ -504,11 +512,17 @@ func (w *walletUnlockerService) restoreAccount(
 	ww *wallet.Wallet, accountIndex int,
 ) *accountLastDerivedIndex {
 	lastDerivedIndex := &accountLastDerivedIndex{}
-	for chainIndex := 0; chainIndex <= 1; chainIndex++ {
+	ranges := [][2]int{
+		{w.rescanRangeStart, w.rescanRangeEnd}, // configurable range for external addresses
+		{0, 20},                                // fixed range for internal ones
```

Comment:
shoudl come from env vars?

Reply:
this is the range for restoring internal addresses. This shouldn't be configurable because the user is not allowed to manage directly these addresses.

The range above is the one related to external address, which is indeed configurable via env var